documentation/general/dash-sonic-hld.md

```diff
@@ -265,6 +266,73 @@ It is possible that a given packet can get a hit in route table and/or mapping t
 
   ![dash-outbound-meter](./images/dash-hld-outbound-meter-pipeline.png)
 
+## 2.5 FastPath
+
+This section captures the Sonic-Dash specifics of FastPath use-case. Detailed document on FastPath is captured here (![FastPath](https://github.com/sonic-net/DASH/blob/main/documentation/load-bal-service/load-balancer-v3.md))
+The following are the salient points and requirements:
+- FastPath kicks in when appliance receives an ICMP redirect that matches an existing unified flow
+- ICMP redirects are expected to be received from source and destination MUXes separately
```

Comment:
This might not be true. The ICMP redirects are sent from one side of the MUX, but there will be 2 ICMP redirect packets, one for each side.

Also, I feel maybe we could simply emit this point? because who sends the redirect packet doesn't really matter, the one matters is we need to receive the packets.

Reply:
omitted the line and its anyways captured in the fastpath doc linked in this section